pkg/catalog/manager/crd.go

```diff
@@ -145,7 +153,10 @@ func mergeLabels(set1, set2 map[string]string) map[string]string {
 
 func (m *Manager) getTemplateVersion(templateName string, namespace string) (map[string]struct{}, error) {
 	//because templates is a cluster resource now so we set namespace to "" when listing it.
-	r, _ := labels.NewRequirement(TemplateNameLabel, selection.Equals, []string{templateName})
+	r, err := labels.NewRequirement(TemplateNameLabel, selection.Equals, []string{templateName})
+	if err != nil {
+		return nil, err
```

Comment:
determined that error being returned has information needed and wrapping is not necessary. it is returning an error string so it does not have a status and has a very specific error message

Reply:
I think this may require more context for someone who is reading logs, such as "failed to get template version [name]: improper label (or include label error)"